packages/dds/merge-tree/src/client.ts

```diff
@@ -1168,8 +1168,24 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 		}
 	}
 
+	/**
+	 * @deprecated - Use searchForTile instead.
+	 */
 	findTile(startPos: number, tileLabel: string, preceding = true) {
 		const clientId = this.getClientId();
 		return this._mergeTree.findTile(startPos, clientId, tileLabel, preceding);
 	}
+
+	/**
+	 * Replaces the findTile API for searching a string for the nearest tile in either direction.
```

Comment:
You'll either have to rewrite this when you remove findTile (since it will refer to a nonexistent function) or you can not refer to findTile here in the first place.

Reply:
same for mergetree